### DISCUSSION TITLE
GitHub account sudden suspension

### DISCUSSION THREAD

**Discussion by HimanshuW4** [community/community#116910] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body
On 26 March, 2024. I received a suspicious activity notification. Which mentioned forcing login and then later changing the password. I try to change the password and the next thing I know my account is suspended for unknown or unclear reasons. Thereafter, I raised the request twice via GitHub and sent them an email as well. Yet there's no update until now. 
I tried to R&D on this but didn't get so far. Just cause, GitHub Community can't help in this as they mentioned here: https://github.com/orgs/community/discussions/58191. Now what am I supposed to do with this? Neither I can check the status of the request nor does GitHub usually reply on time due to high volumes. I have a free account, but it doesn't mean that it's not worth it. 
Everything got scrambled because this happened at an unusual time. Today, I have seen that I am not alone who is facing this issue. There are others as well, who have encountered the same issue and can't do anything about it and everyone is seeking help including me.

**Comment by HimanshuW4** [comment]
Even though github community might not help because it's their ground to settle in issues with account suspension. I hope Github officials must closed this discussion sooner or later.

**Reply by HimanshuW4** [reply]
@gensart-x have you got any reply for Github regarding your account suspension?

**Reply by gensart-x** [reply]
Hi, thanks for the mention.

About the update of my suspension, all I can say is nope. Nothing ever come to my email since the suspension until now. I can't expect much and almost giving up all of my contents inside.

Now I'm already do follow up via email to my current ticket about the suspension update, but still, nothing.

I'm creating a new space for my old account to this account.

To be honest, I'm really hopeless, and weary for a support quality like this, knowing that I trust GitHub to put my work there. Now I even registering another VC like GitLab and trying to begin transition if possible, or find a possible way to do synchronization between GitHub n GitLab.

**Reply by HimanshuW4** [reply]
> Hi, thanks for the mention.
> 
> 
> 
> About the update of my suspension, all I can say is nope. Nothing ever come to my email since the suspension until now. I can't expect much and almost giving up all of my contents inside.
> 
> 
> 
> Now I'm already do follow up via email to my current ticket about the suspension update, but still, nothing.
> 
> 
> 
> I'm creating a new space for my old account to this account.
> 
> 
> 
> To be honest, I'm really hopeless, and weary for a support quality like this, knowing that I trust GitHub to put my work there. Now I even registering another VC like GitLab and trying to begin transition if possible, or find a possible way to do synchronization between GitHub n GitLab.

Hi, thanks for reverting back.
You know it's same here no reply until now even after ticket raised. No other mails received so far and no update until so far.
And thereafter I got blamed at work that's it's my fault that your account is suspended and they mentioned that what happened if organisation data got leaked. Even though everything was logged out automatically. I was on the verge of being fired from Job. And the reason just pop up out a nowhere!

**Reply by gensart-x** [reply]
Hi,

Aw that seem bad in your situation, I hope you're having good adaptation on your work with this suspension.

But yeah, the only thing we can do after all is just to wait for an update in our ticket. Because, after I read some of discussions here, community can't help but just advising to wait.

I even beginning to forgetting that haha, 😆.

**Reply by HimanshuW4** [reply]
Getting a new one won't be an easy task, it will take time. I hope the GitHub team replies soon enough with a resolution. And I agree with you that in discussions and replies from GitHub officials in the community can't do anything about the suspension. We must at least know the reason or else the status check option should be there so, that it doesn't have to be helpless for now. It's not just the data, that is gone, it's actually much more than that.

**Reply by harshm2601** [reply]
@gensart-x  if these things are happening with many Users , why github hasn't  taken any actions yet  or atleast they should give a notice regarding this issue. I had student developer pack which allowed me to use co pilot and now i can't  even finish one line of code XD.

**Comment by harshm2601** [comment]
I am Having Same issue , kindly reply me if your your account is recovered.

**Comment by HimanshuW4** [comment]
> I am Having Same issue , kindly reply me if your your account is recovered.

You know everyone including me is waiting for their reply. It happens for the first time to most of the users. And people are unaware about why exactly it happened. Regrading the recovery, it will take time don't know how much. Yet sooner or later we get something out of it. Until then we can't check the status nor we can have chat or call GitHub. In that case only two things are left to do at current. At first, raise an appeal request to GitHub and second, stop worrying about reply and move on with new account until and unless our accounts got reinstated. I suppose the quantity of reinstate must be a lot that's why there's no update so far. So until that time we're here with you and if there's any update then I will update my thread.

**Reply by Naman-Acharya** [reply]
Yeah I don't know how but most of users faced similar issue even my account got suspended without any reason. Might be this is because of malware as per the current news but still unaware about it. I hope we get our account back as soon as possible because specially most of my work was on Github and with Github Copilot and all the work was suddenly stopped because of this incident. 

My Ticket No: #2737022

**Comment by ghost** [comment]
@HimanshuW4 
Did you get a response? Mine got suspended too, it has been more than a month and still no response.
ticket: #2719793

**Reply by HimanshuW4** [reply]
No response yet, I am still waiting for it. And the account is still suspended!! Done know if the response ever going to come or not.

Although mail related to GitHub are still coming but not relevant to account suspension or anything related to that.

**Reply by gensart-x** [reply]
hopeless bro, I've been too since late March.

**Reply by HimanshuW4** [reply]
Same here brother. There has been no update so far. It's time to move on from now on.

**Comment by HimanshuW4** [comment, marked as answer]
@everyone I have to inform you all that my old GitHub account has been reinstated. I don't know how, but all of a sudden I received the password reset email in my inbox, and the next thing I knew my account is live again. So, yes it's an update to everyone who commented and asked for an update. Here we go. It's a miracle.
